sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/GatewayAddressCacheTest.java

```diff
@@ -800,7 +800,7 @@ public void tryGetAddresses_SuboptimalMasterPartition_Stale_DoRefresh() throws E
                         "/dbs",
                         new Database(), new HashMap<>());
 
-        PartitionKeyRangeIdentity partitionKeyRangeIdentity = new PartitionKeyRangeIdentity("M");
+        PartitionKeyRangeIdentity partitionKeyRangeIdentity = new PartitionKeyRangeIdentity(createdCollection.getResourceId(), "M");
```

Comment:
I think we should leave the collectionRid as null in the tests, it helped to capture some regressions

Reply:
Good point, done.